### PR TITLE
[DOC](storage policy) add cold and hot separation docs

### PR DESCRIPTION
**Pull request by deardeng**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

add cold and hot separation docs

### CODE REVIEW

**Review by morningman — COMMENTED** [review]
We also need a document to introduce this feature, and best practice of this feature for user. You can refer to 

```
https://doris.apache.org/zh-CN/docs/dev/advanced/cache/partition-cache
```

**Comment by deardeng** [comment]
> We also need a document to introduce this feature, and best practice of this feature for user. You can refer to
> 
> ```
> https://doris.apache.org/zh-CN/docs/dev/advanced/cache/partition-cache
> ```

I added it.

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE.md [hunk 1]
```diff
@@ -50,6 +50,22 @@ ALTER RESOURCE 'spark0' PROPERTIES ("working_dir" = "hdfs://127.0.0.1:10000/tmp/
 ```sql
 ALTER RESOURCE 'remote_s3' PROPERTIES ("s3_max_connections" = "100");
 ```
+3. 修改冷热分离S3资源相关信息
+- 支持修改项
+  - s3_max_connections s3最大连接数，默认50
```
Comment (morningman) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE.md:55]: 所有参数需要用反引号引起来，否则在在 markdown中展示，会识别为两个下划线括起来的 “斜体”

`s3_max_connections`

Comment (morningman) [reply docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE.md:55]: 另
1. 需要在 `docs/sidebars.json` 中添加所有新增文档的链接，否则网站不会显示。
2. `docs/zh-CN/docs/advanced/cold_hot_separation.md` 需要有英文版本。

Comment (deardeng) [reply docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-RESOURCE.md]: fixed. please review again.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**